### ISSUE TITLE
User Dashboard New Project Button Exposed at All Time

### ISSUE THREAD

**Issue by jiajasper** [playcanvas/editor#820]
The new project button should be exposed at all time instead of just on the project tab

<img width="595" alt="image" src="https://user-images.githubusercontent.com/30747200/177558130-c912ef16-0bef-4e89-8256-1b97d178f059.png">

**Comment by willeastcott** [comment]
I disagree with this one. I don't want the New Project button on the Project dashboard page. Closing.
